internal/distro/rhel8/imagetype.go

```diff
@@ -374,7 +374,7 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 
 	if osc := customizations.GetOpenSCAP(); osc != nil {
 		// only add support for RHEL 8.7 and above. centos not supported.
-		if !t.arch.distro.isRHEL() || common.VersionLessThan(t.arch.distro.osVersion, "8.7") {
+		if common.VersionLessThan(t.arch.distro.osVersion, "8.7") {
```

Comment:
You should also adjust the comment above the line, since it says `centos not supported`